addons/main/weapons/CfgWeapAR15.hpp

```diff
@@ -122,6 +135,8 @@ class hlc_wp_mod727: hlc_M16Base {
     };
 };
 class hlc_wp_mod733: hlc_M16Base {
+    ACE_barrelTwist = 228.6;
```

Comment:
As far as I can tell, it was a 1:7 barrel twist they used. If you have sources that say a 1:9 twist, can you share them?

```suggestion
    ACE_barrelTwist = 177.8;
```